docs/try-flink/python_table_api.md

```diff
@@ -23,161 +23,9 @@ specific language governing permissions and limitations
 under the License.
```

Comment:
If you are moving the content, please just delete this page and setup a redirect.